runelite-client/src/main/java/net/runelite/client/plugins/banktags/tabs/TabInterface.java

```diff
@@ -85,6 +85,22 @@
 	private static final int MARGIN = 1;
 	private static final int SCROLL_TICK = 500;
 
+	private int BANK_CONTAINER_ID = 0;
```

Comment:
only constants should be upper_case, and also you can just reference WidgetInfo.BANK_CONTAINER.getId() directly in the doSearch method, so do that and remove this variable and code on line 165